sdk/servicebus/azure-servicebus/tests/perf_tests/T1_legacy_tests/_test_base.py

```diff
@@ -66,8 +68,14 @@ def __init__(self, arguments):
             shared_access_key_name=connection_props['shared_access_key_name'],
             shared_access_key_value=connection_props['shared_access_key']
         )
+
+    async def global_setup(self):
+        await super().global_setup()
         queue = Queue(max_size_in_megabytes=40960)
         self.mgmt_client.create_queue(self.queue_name, queue=queue)
+
+    async def setup(self):
+        await super().setup()
         self.queue_client = self.service_client.get_queue(self.queue_name)
```

Comment:
NIT: Does getting a queue_client make a network call, or is it a client-only operation?  If it's client-only, the pattern we've typically been using is to put it into the ctor.  In short, ctor is for any code which doesn't make a network call, then async setup/global_setup are for code which does make network calls or is otherwise slow.

Reply:
Yes - for T1 this does make a network call - and needs to happen after the queue is created in `global_setup`